saas_demo_test/README.rst

```diff
@@ -8,6 +8,11 @@
 
 This addon is not meant to be used.
 
+Roadmap
+=======
+
+* TODO: add a test repository to the addons path in advance, since we cannot reload the odoo in tests.
```

Comment:
Wrong. ToDO is something you need to do. But we don't need to add a repository to addons path (now)